src/applications/disability-benefits/new-686/config/generic-schema.js

```diff
@@ -31,13 +32,17 @@ export const genericSchemas = {
   genericNumberInput: {
     type: 'string',
     maxLength: 50,
-    pattern: numberOnlyPattern,
+    pattern: numberAndDashPattern,
```

Comment:
Following the comment above, it might be good to have a separate and distinct `phoneNumber` property here and leave `genericNumberInput` how it was, unassociated with the `numberAndDashPattern`.

Reply:
totally agree, updated